src/app_unit.rs

```diff
@@ -289,6 +289,92 @@ impl Au {
     }
 }
 
+#[cfg(feature = "num_traits")]
+impl num_traits::ToPrimitive for Au {
+    // Float conversions
```

Comment:
I think these comments aren't adding too much. It's pretty obvious what the conversion is by looking at the type signature.